server/routerlicious/packages/services/src/redis.ts

```diff
@@ -31,6 +31,10 @@ export class RedisCache implements ICache {
             Lumberjack.error("Error with Redis", undefined, err);
         });
     }
+    public async delete(key: string): Promise<boolean> {
+        const result = await this.client.del(this.getKey(key));
+        return result === 1;
```

Comment:
I would not use `result === 1` as a check of success of not. Because if a key already evicted/expired/not in cache, this would result 0 but the command itself is success.

If you want to indicate the results, we can use try/cache instead.

Reply:
changed to try/catch